de.metas.endcustomer.mf15/de.metas.endcustomer.mf15.base/src/main/jasperreports/de/metas/docs/purchase/inout/report.properties

```diff
@@ -8,6 +8,7 @@ vonuns = von uns
 showfooter=Y
 showdetailsfooter=Y
 details_product_overflow=N
+showdescription=Y
```

Comment:
And for this particular case, please also mention which order-line field needs to be visible for it all to make sense